plugins/modules/content_export_library.py

```diff
@@ -103,7 +103,7 @@
     organization: "Default Organization"
     destination_server: "airgapped.example.com"
     incremental: true
-    from_history_id: "12345"
+    from_history_id: 12345
```

Comment:
funnily, this was the only example using a string, the others were using ints ;)